libconnman-qt/networkmanager.cpp

```diff
@@ -530,12 +530,7 @@ NetworkService* NetworkManager::defaultRoute() const
 
 NetworkTechnology* NetworkManager::getTechnology(const QString &type) const
 {
-    if (m_technologiesCache.contains(type))
-        return m_technologiesCache.value(type);
-    else {
-        qDebug() << "Technology " << type << " doesn't exist";
-        return NULL;
-    }
+    return m_technologiesCache.value(type);
```

Comment:
Is it okay to return an default-constructed NetworkTechnology object? Doing

> return  m_technologiesCache.value(type, NULL);

or equivalent would preserve the old functionality.